brambling/utils/model_tables.py

```diff
@@ -713,3 +727,9 @@ def housing_categories(self, obj):
 
     def balance(self, obj):
         return format_money(obj.balance or 0, self.event.currency)
+
+    def completed_date(self, obj):
+        if obj.completed_date:
+            return obj.completed_date.strftime("%Y-%m-%d %H:%M")
```

Comment:
This is fine, for now, given that the housing date is handled similarly. But this does need to be wrapped in a timezone context manager, like what I think you did for the finances csv?